controllers/contacts.js

```diff
@@ -0,0 +1,102 @@
+const { object, string, number, date, InferType, array } = require("yup");
+
+//let contactsMock = require('./mockData')
+let contactsMock1 = [
```

Comment:
It is a literal constant, so please use UPPER_CASE
```suggestion
let CONTACT_MOCK = [
```